cmd/dra-example-kubeletplugin/main.go

```diff
@@ -159,18 +162,32 @@ func StartPlugin(ctx context.Context, config *Config) error {
 		return fmt.Errorf("path for cdi file generation is not a directory: '%v'", err)
 	}
 
-	driver, err := NewDriver(ctx, config)
-	if err != nil {
-		return err
-	}
+	var driver *driver
+	{
+		// A new context only for NewDriver. We don't want to use an
+		// already-done context after that finishes.
```

Comment:
Does this comment still make sense?

I have the nagging feeling that this code at some point used `signal.NotifyContext(context.Background(), ...`, but I don't see it in the PR history.